src/huggingface_hub/inference/_generated/types/text_classification.py

```diff
@@ -14,18 +14,14 @@
 
 @dataclass
 class TextClassificationParameters(BaseInferenceType):
-    """
-    Additional inference parameters for Text Classification.
+    """Additional inference parameters
+    Additional inference parameters for Text Classification
```

Comment:
(nit) the duplicate docstring comes from [huggingface/huggingface.js/tasks/text-classification/spec/input.json#L13](https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/tasks/text-classification/spec/input.json#L13)  and [huggingface/huggingface.js/tasks/text-classification/spec/input.json#L20](https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/tasks/text-classification/spec/input.json#L20), same for other tasks.

Reply:
Let's remove it then (we can still merge this PR first)

Reply:
```
		"parameters": {
			"$ref": "#/$defs/TextClassificationParameters"
		}
```
seems enough

Reply:
i'm opening a PR now to remove it!